lib/extensions/user_notifications_extension.rb

```diff
@@ -14,13 +14,17 @@ def chat_summary(user, opts)
         .joins(
           "INNER JOIN user_chat_channel_memberships uccm ON uccm.chat_channel_id = chat_channels.id",
         )
-        .where(uccm: { following: true, user_id: user.id })
-        .where(<<~SQL)
-          (cm.user_id = #{user.id} OR chat_channels.chatable_type = 'DirectMessageChannel') AND
+        .where(<<~SQL, user_id: user.id)
+          uccm.user_id = :user_id AND
           (uccm.last_read_message_id IS NULL OR chat_messages.id > uccm.last_read_message_id) AND
-          (uccm.last_unread_mention_when_emailed_id IS NULL OR chat_messages.id > uccm.last_unread_mention_when_emailed_id)
+          (uccm.last_unread_mention_when_emailed_id IS NULL OR chat_messages.id > uccm.last_unread_mention_when_emailed_id) AND
```

Comment:
I am surprised there is no spec that needs to be changed for this? Maybe we are missing one?